doc/usecases/visitor_chat.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Visitor chat
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** 
+  - Owner: App User who owns the map
+  - Visitor: App User who visits the map
+- **Brief:** The user can interact with a visitor chat.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The visitor has opened the app and selected a map.
+- **Main success scenario:**
+  - The visitor is automatically greeted by the chat. 
+  - The visitor can interact with the owner and other current visitors via the chat.
+  - The owner can view the chat and interact with all visitors.
```

Comment:
It would be nice for the owner to mark a comment left by another user as helpful. A comment marked as helpful should also award points for the free membership of the author.